ansible/inventories/production/group_vars/all/vars.yml

```
@@ -25,6 +25,7 @@ catalog_pycsw_db_host: "{{ vault_catalog_pycsw_db_host }}"
 catalog_pycsw_db_name: "{{ vault_catalog_pycsw_db_name }}"
 catalog_pycsw_db_pass: "{{ vault_catalog_pycsw_db_pass }}"
 catalog_pycsw_db_user: "{{ vault_catalog_pycsw_db_user }}"
+catalog_validate_redirect: true
```

Since this is specific to the smoke playbook, I would prefix it with that. Also, follow the existing convention from the `smoke_enable_cdn_checks` variable.
```suggestion
smoke_catalog_tracking_check: true
```

---

I think we want to pivot then to something more like this newest commit, using different variables to hit the tracking URL and then when the instance is readonly then validate that the redirect is working...